spec/Section 3 -- Type System.md

```diff
@@ -343,9 +343,17 @@ client-specific primitive for time. Another example of a potentially useful
 custom scalar is `Url`, which serializes as a string, but is guaranteed by
 the server to be a valid URL.
 
+When defining an additional scalar, GraphQL systems should use the `@specified`
+directive to provide a URL pointing to a human-readable specification of the
+data format, serialization, and coercion rules for the scalar. For example, a
+GraphQL system providing a `UUID` scalar might link to RFC 4122, or some
+document defining a reasonable subset of that RFC. If a specification URL is
+present, systems must conform to the described rules. Built-in scalar types
+should not provide a URL.
+
 ```graphql example
-scalar Time
-scalar Url
+scalar UUID @specified(by: "https://tools.ietf.org/html/rfc4122")
+scalar URL @specified(by: "https://tools.ietf.org/html/rfc3986")
```

Comment:
@leebyron I changed the capitalization here from `Url` to `URL` because `Uuid` looked particularly gross, and the built-in is already `ID` not `Id`. It's a minor editorial thing but it's not really related to this RFC so I wanted to make sure it was ok.

Reply:
Seems totally fine to me